docs/install-advanced.md

```diff
@@ -0,0 +1,130 @@
+# Advanced Install Guide
+
+In certain situations, users are given a Kubernetes cluster with tighter security constraints such as
+
+- only being able to deploy applications to a certain namespace
+- running applications in an RBAC-enabled cluster
+
+This document explains some of these situations as well as how it can be handled using draft.
+
+## Running Draftd with RBAC enabled
+
+To install draftd in a cluster with RBAC enabled, a few additional steps are required to grant tiller access to deploy in namespaces.
+
+[Helm's documentation] on setting up Tiller in RBAC-enabled clusters is the best document on the subject, but for new users who just want to get started, let's create a new service account for tiller with the right roles to deploy to the `default` namespace.
```

Comment:
I think you might be missing a link here ^

Reply:
indeed I am. Nice catch.